src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -201,8 +201,9 @@ public ProcessWrapper launchTranscode(
 		}
 
 		// Streaming support
-		cmdList.add("-movflags");
-		cmdList.add("+faststart");
+//		cmdList.add("-movflags");
+//		cmdList.add("frag_keyframe+empty_moov");
+		
```

Comment:
@Sami32 Are you just dumping this at google translate? I really don't understand what you're trying to say some times, like above.

Google translate (and similar services) isn't usable imo, try translating a sentence from French to English and then back to French and see if you can understand it.